src/components/NumberIcon/NumberIcon.tsx

```diff
@@ -67,7 +81,18 @@ export const NumberIcon = ({
       variant={variant === 'base' ? 'base' : 'white'}
       {...other}
     >
-      {number}
+      {incomplete && numberIconTitle ? (
+        <Icon
+          className={styles['number-icon__icon']}
+          color={EdsThemeColorIconNeutralSubtle}
+          name="circle"
+          purpose="informative"
+          size={size === 'sm' ? '0.25rem' : '0.5rem'}
+          title={numberIconTitle}
+        />
+      ) : (
+        number
```

Comment:
What does this look like if there's no number?

Reply:
Seemed to make sense building that into this rather than a user having to define that in their component each time.

Reply:
Yep, that sounds right, I just wanted to make sure. Could you add that as a comment on the `number` prop? And maybe in the component comment too. People might build that themselves not realizing that functionality included in the component.